src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditorTemplate.html

```diff
@@ -0,0 +1,121 @@
+<div tabindex="-1" class="border-radius-editor">
+    <header>
+        <ul class="button-bar">
+          <li id="allCorners" title="All Corners">
+              <a href="#" tabindex="0">All Corners</a>
```

Comment:
We'll need to localize all user visible strings.

Reply:
@flukeout @humphd I noticed there are so many files in the src/nls folder for storing all localized strings. Since I need localize all those user visible strings, can anyone tell me where I should save those string key value pairs to?